TestPlan.md

```diff
@@ -0,0 +1,53 @@
+#Test plan for Some User Experience (SUX)
+
+## TABLE OF CONTENTS</br>
+Introduction</br> 
```

Comment:
Instead of line breaks, use markdown syntax.

`###Scope`
`##Testing Strategy`
`* Unit Testing`
`* System and Integration Testing`